tasks/main.yml

```diff
@@ -21,6 +23,19 @@
   command: "openssl dhparam -out /etc/prosody/certs/dh-{{ prosody_dhparam_length }}.pem {{ prosody_dhparam_length }} creates=/etc/prosody/certs/dh-{{ prosody_dhparam_length }}.pem"
   notify: restart prosody
 
+- name: Get latest external prosody modules
+  hg:
+    repo: https://hg.prosody.im/prosody-modules/
+    dest: /usr/share/prosody-modules
+
+- name: Enable external modules
```

Comment:
Please use `plugin_paths` in the prosody config instead of linking the modules in the prosody modules path.
Maybe change `/usr/share/prosody-modules/` to `/usr/share/prosody-external-modules/` to make this clear